scripts/building/vscode_pytest/run_pytest_script.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
I understand this script can be used for vscode run test quickly, which is great, BUT, maybe we can put it another place? for me, `scripts/building` is not a good place for such usage.

Reply:
also, maybe add a README on how to use this?